### PR TITLE
Update some instances of AdoptOpenJDK to Adoptium

### PR DESCRIPTION
**Pull request by pshipton**
I didn't update the jenkins code, or the vendor used in the API for excluding tests.

### CODE REVIEW

**Comment by pshipton** [comment]
The copyright failure is expected as .github/ISSUE_TEMPLATE/test-suite-failure.md doesn't have a copyright.

**Comment by keithc-ca** [comment]
> The copyright failure is expected as .github/ISSUE_TEMPLATE/test-suite-failure.md doesn't have a copyright.

I created #14264, which, among other things, would allow for a copyright notice.

File: doc/processes/labels.md [hunk 1]
```diff
@@ -102,5 +102,5 @@ End User issues
 To indicate an issue has been raised by an end user or affects the User-experience rather than being a developer-centric issue.
 
 * `userRaised`
-* `svtRaised`: Indicate an issue was raised for an AdoptOpenJDK 3rd party issue or the OpenJ9 system verification test (svt).
+* `svtRaised`: Indicate an issue was raised for an Adoptium 3rd party issue or the OpenJ9 system verification test (svt).
```
[the remark this answers is not in the document]
Comment (pshipton) [reply doc/processes/labels.md]: Updated.

**Review by keithc-ca — APPROVED** [review]

**Review by llxia — APPROVED** [review]
LGTM

**Comment by llxia** [comment]
will forgo the PR build as this is a documentation change.

**Pull request merged**